From 74230634260455673dcea330bb95a53c0b6f6f6a Mon Sep 17 00:00:00 2001
From: Davor Joja <davorjoja@logicbricks.com>
Date: Wed, 5 Nov 2014 16:51:03 +0100
Subject: [PATCH 266/290] gpu: drm: xylon: Removed unused function argument

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Removed unused struct device *dev function argument.

Signed-off-by: Davor Joja <davorjoja@logicbricks.com>
Tested-by: Christian Kohn <christian.kohn@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 476013203cd0cccf70901f4a7297fb12e74055d5)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/xylon/xylon_logicvc.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/xylon/xylon_logicvc.c b/drivers/gpu/drm/xylon/xylon_logicvc.c
index 127a67f..57674de 100644
--- a/drivers/gpu/drm/xylon/xylon_logicvc.c
+++ b/drivers/gpu/drm/xylon/xylon_logicvc.c
@@ -697,8 +697,7 @@ void xylon_cvc_disable(struct xylon_cvc *cvc)
 			xylon_cvc_layer_disable(cvc, i);
 }
 
-static int xylon_parse_hw_info(struct device *dev,
-			       struct device_node *dn, struct xylon_cvc *cvc)
+static int xylon_parse_hw_info(struct device_node *dn, struct xylon_cvc *cvc)
 {
 	int ret;
 	const char *string;
@@ -970,7 +969,7 @@ struct xylon_cvc *xylon_cvc_probe(struct device *dev, struct device_node *dn)
 		 LOGICVC_MINOR_REVISION_MASK),
 		 ((ip_ver & LOGICVC_PATCH_LEVEL_MASK) + 'a'));
 
-	ret = xylon_parse_hw_info(dev, dn, cvc);
+	ret = xylon_parse_hw_info(dn, cvc);
 	if (ret)
 		return ERR_PTR(ret);
 
-- 
2.0.2

